channels/ro.m3u

```diff
@@ -129,8 +129,6 @@ http://89.38.8.130:39443
 http://82.76.40.76/utvedge/utvlive/playlist.m3u8
 #EXTINF:-1 tvg-id="UTV.ro" tvg-name="UTV" tvg-country="RO" tvg-language="Romanian" tvg-logo="" group-title="Music",UTV (576p) [Offline]
 http://edge76.rcs-rds.ro/utvedge/utvlivehls/playlist.m3u8
-#EXTINF:-1 tvg-id="UTV.ro" tvg-name="UTV" tvg-country="RO" tvg-language="Romanian" tvg-logo="" group-title="Music",UTV (576p) [Offline]
-https://edge126.rcs-rds.ro/utvedge/utvhq.stream/index.m3u8
```

Comment:
Having a long ass URL for nothing I hope.

https://edge214.rcs-rds.ro/V4-9812783-1d88fea095f96f4dd5d1c13e8e877693-1538107471-0-1630225960-411-utv/utv/utv-abr.m3u8?quality=hq&t=00000000000000000000000000000000&t4=V4-9812783-1d88fea095f96f4dd5d1c13e8e877693-1538107471-0-1630225960-411-utv